scripts/generate-upstream-patch.sh

```diff
@@ -52,7 +53,8 @@ generate_upstream_patch() {
 	echo "fetch upstream_$TARGET"
 	git fetch -q "upstream_$TARGET"
 
-	local tmp_dir=$(mktemp -d)
+	local tmp_dir
+	tmp_dir=$(mktemp -d)
```

Comment:
Really? That's an odd convention imho but if it makes the bot happy then okay :D

Reply:
Yea: https://www.shellcheck.net/wiki/SC2155